src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -609,7 +612,19 @@ private void OnDiscoveryMessageReceived(ITestDiscoveryEventsHandler2 discoveryEv
             OnDiscoveryAbort(discoveryEventsHandler, ex, false);
         }
     }
-
+    private static bool BlameIsEnabled()
+    {
+        string? s = RunSettingsManager.Instance.ActiveRunSettings.SettingsXml;
```

Comment:
`s` isn't really meaningful, please rename into `runSettingsXml` or `runSettingsString`. Or you could also remove the variable and pass it directly to `GetDataCollectionRunSettings` call.